src/Catalyst.Cli.Tests/UnitTests/Commands/Response/AddFileToDfsResponseTests.cs

```diff
@@ -73,7 +73,7 @@ public void AddFileToDfsResponse_Finished_Can_Get_Output()
             };
 
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var addFileToDfsCommand = new AddFileCommand(null, commandContext, _logger);
+            new AddFileCommand(null, commandContext, _logger);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'AddFileCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)